Soccar [UI]/src/nl/socnet/message/handler/ChatMessageHandler.java

```diff
@@ -3,13 +3,18 @@
 import io.netty.buffer.ByteBuf;
 import javafx.application.Platform;
 import nl.soccar.gamecommuncation.util.ByteBufUtilities;
+import nl.soccar.library.Player;
+import nl.soccar.library.Session;

```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'nl.soccar.library.Session'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUselessImportCheck)